src/App/Homepage/Narrowscreen/NarrowFacebookTicker.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const NarrowTicker = () => (
```

Comment:
please change this to be called FacebookFeed, so that I know what it is, a ticker is something like what shows your stock prices and is moving, thanks